src/runtimeResolver.js

```diff
@@ -148,6 +148,12 @@ RuntimeResolver.prototype.resolveRuntimes = function(){
     if (!runtime.capabilities) {
       runtime.capabilities = {};
     }
+
+    if (_.get(that, "config.log.browser.level")) {
```

Comment:
I really want to have this code in the plugin, we could extend the plugin API if necessary ?